src/Controls/tests/UITests/Tests/Issues/Issue18111.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18111 : _IssuesUITest
+	{
+		public Issue18111(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "Setting MaximumTrackColor on Slider has no effect";
+
+		[Test]
+		public void SettingMaximumTrackColorOnSliderWorks()
+		{
+			App.WaitForElement("WaitForSliderControl");
```

Comment:
Use:

`this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Windows });`

If you want to run the test only on iOS and macOS.